CHANGELOG.md

```diff
@@ -47,6 +47,10 @@ Synchronized Realms require a server running Realm Object Server v2.0 or higher.
 * Fix unmanaged object initialization when a nested property type returned
   `false` from `Object.shouldIncludeInDefaultSchema()`.
 * Don't clear RLMArrays on self-assignment.
+* The Objective-C and Swift `create(_:, value: update:)` APIs now
```

Comment:
It's a breaking change, but not a breaking API change. If you prefer it be in the first section, I can move it back.